build.gradle

```diff
@@ -44,6 +44,7 @@ apply from: 'gradle/formatting.gradle'
 apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.rest-test'
 apply plugin: 'opensearch.pluginzip'
+apply plugin: 'opensearch.repositories'
```

Comment:
Why we need this?

Reply:
it's required to get some dependencies for Core OpenSearch, for instance for us it was failing without this plugin on getting proper lucene version, please see details here https://github.com/opensearch-project/OpenSearch/issues/3455

Reply:
Is this is the best way to get the lucene, what about the snapshot provided in the github issue?

`maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }`

Reply:
both approaches recommended by Core team, my guess they are equivalent

Reply:
This post provides good explanation - https://github.com/opensearch-project/OpenSearch/issues/2500#issuecomment-1072046551